cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller_data_types.h

```diff
@@ -64,9 +68,140 @@ namespace cob_cartesian_controller
         unsigned int move_type;
         MoveLinStruct move_lin;
         MoveCircStruct move_circ;
+        ProfileStruct profile;
     };
 
-}//namespace
 
 
+    class PathArray{
```

Comment:
maybe we can move these classes and the implementations into the cartesian_controller_utils?